app/_src/production/install-kumactl.md

```diff
@@ -56,7 +56,7 @@ and extract the archive with `tar -xvzf {{ site.mesh_helm_install_name }}-{{ pag
 
 Add the `kumactl` executable to your path:
 ```
-cd {{site.mesh_product_name | downcase }}-{{ page.version_data.version }}/bin
+cd {{site.install_archive_name }}-{{ page.version_data.version }}/bin
```

Comment:
please use `mesh` as a prefix (useful for docs.konghq.com)

Reply:
done